src/Sql/SqlPgsql.php

```diff
@@ -178,4 +180,44 @@ public function getPasswordFile()
     {
         return $this->password_file;
     }
+
+    /**
+     * Get schema name.
+     *
+     * @return string
+     */
+    public function getSchemaName()
+    {
+        $schema = 'public';
+        $dbSpec = $this->getDbSpec();
+        if (!empty($dbSpec['prefix'])) {
```

Comment:
I think `!empty` will be a warning on a certain versions of PHP rather than `isset`.

Reply:
Not according to docs https://www.php.net/empty